notebooks/01_tabular_data_exploration.ipynb

```diff
@@ -182,12 +187,10 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "lines_to_next_cell": 2
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "adult_census.hist(figsize=(20, 10));"
+    "adult_census.hist(figsize=(20, 10))"
```

Comment:
This was intentional. Isn't there a way to configure yapf not to remove trailing semi colons?

Reply:
good catch. I will check. Otherwise `_ = ...` will make the trick.

Reply:
Indeed we can always do that.

Reply:
so apparently, you have to use `# yapf: disable` so I would use `_ = ...` then.